### ISSUE TITLE
Helper Tool installs TablePlus, after disabling Helper Tool and removing Proxyman, TablePlus is still there

### ISSUE THREAD

**Issue by idevelop-app** [ProxymanApp/Proxyman#1405]

**Comment by NghiaTranUIT** [comment]
I'm sorry, but the Proxyman Helper Tool doesn't install the TablePlus app.

Proxyman Helper Tool just helps you to automatically override/revert the HTTP Proxy when the app opens/quits.

Maybe TablePlus is already installed on your Macbook/system. 

Document: https://docs.proxyman.io/basic-features/proxy-setting-tool

**Comment by idevelop-app** [comment]
> I'm sorry, but the Proxyman Helper Tool doesn't install the TablePlus app.
> 
> Proxyman Helper Tool just helps you to automatically override/revert the HTTP Proxy when the app opens/quits.
> 
> Maybe TablePlus is already installed on your Macbook/system.
> 
> Document: https://docs.proxyman.io/basic-features/proxy-setting-tool

This is a brand new installation, no other app is installed, no security team is involved, as soon as I added the Proxy Helper Tool I got a notification in macOS Ventura RC that the background process TablePlus Inc has been installed and will be running in the background, I just double check it again, by installing a brand new Virtual Machine replicating the whole installation process, and exactly the same happened

**Comment by idevelop-app** [comment]
Ok, that makes sense, when I disable the helper, the helper tool will be removed as well, we are ok with that, nonetheless the TablePlus Inc background process is not removed from Login Items (look at the print screen), even If I fully remove the Proxyman app, TablePlus Inc will remain in the Login Items

<img width="710" alt="image" src="https://user-images.githubusercontent.com/110728525/196990250-068b1c38-17a4-4acd-a163-af5be2e09be2.png">

**Comment by idevelop-app** [comment]
Even on a brand new installation, after having Disabled the Helper Tool, and Removed Proxyman from the system, the TablePlus Inc. process keeps installed and running:

<img width="1862" alt="image" src="https://user-images.githubusercontent.com/110728525/196991542-6aac1b4c-ee0b-4f61-b18d-2fbbc0d98291.png">

**Comment by idevelop-app** [comment]
> Yes, removing the app doesn't remove the Helper Tool. You have to remove it manually. The same thing happens with Paste, and iStat.
> 
> I will update to macOS Ventura and fix the following issue
> 
> * Show the Proxyman app icon, instead of the generic app icon
> * Double check why Proxyman appears on the Login Item.

Correct, first I disabled the Helper Tool from within Proxyman, successfully, then when I saw the TablePlus Inc. process still installed and running, I wanted to check if by removing Proxyman will help, it didn't.

**Comment by NghiaTranUIT** [comment]
Update: on macOS Ventura, Proxyman Helper Tools is added to the "Allow in Background" section, even though Proxyman only works in the foreground.

It's added due to the Helper Tool.

There is a discussion on this misleading issue on Twitter: https://twitter.com/siracusa/status/1583914093437935616?s=46&t=C1XB91IkDFkG697Ab8T9Dg

**Comment by NghiaTranUIT** [comment]
@idevelop-app Good news here: I've managed to fix the following things:
- Use the Proxyman icon
- Show Proxyman app name, instead of TablePlus Inc
- Able to remove the app from the "Run in the Background" section if you uninstall the Helper Tool.

<img width="473" alt="Screenshot 2022-10-24 at 17 09 55" src="https://user-images.githubusercontent.com/5878421/197530484-0f897b6f-7905-4d65-bdc3-86d2fc8abebe.png">

---
I will test and send you a Beta build soon. The fix is also included in the next release 👍

**Comment by idevelop-app** [comment]
Great news, thank you

**Comment by NghiaTranUIT** [comment]
@idevelop-app Here is the Beta build: https://proxyman.s3.us-east-2.amazonaws.com/beta/Proxyman_3.11.0_New_Helper_Tool.dmg

When the app is launched, you will prompt to update the Helper Tool 1.4.0 👍 

<img width="752" alt="Screen Shot 2022-10-25 at 10 11 32" src="https://user-images.githubusercontent.com/5878421/197673499-f699f614-edad-44d3-b2e0-5a86e61f8a3c.png">

However, the Login Item + Allow in the Background doesn't properly update it due to the caching mechanism (I believe that it's a bug on macOS Ventura). 

To fix it:
1. Open Terminal app -> Run: `sftool resetbtm`
2. Restart the macOS
3. Done. The new Icon + App name will be properly displayed 👍 

If it's the first time you've installed Proxyman (fresh macOS), it always shows the correct one.

**Comment by jankkm** [comment]
> 1. Open Terminal app -> Run: `sftool resetbtm`
> 2. Restart the macOS
> 3. Done. The new Icon + App name will be properly displayed 👍

There is a typo. The command is `sfltool resetbtm`

**Comment by NghiaTranUIT** [comment]
Thanks. It's fixed 👍

**Comment by newadventure079** [comment]
Can we get the docs updated to say `sfltool resetbtm`?  I came here searching for why it was a command that wasn't found.

https://docs.proxyman.io/basic-features/proxy-setting-tool  At the bottom of the page

**Comment by NghiaTranUIT** [comment]
It's fixed @newadventure079 👍
